packages/schema/bind/src/bindings/wasm-rs/types.ts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const baseTypes = {
+  UInt: "UInt",
+  UInt8: "UInt8",
+  UInt16: "UInt16",
+  UInt32: "UInt32",
+  UInt64: "UInt64",
+  Int: "Int",
+  Int8: "Int8",
+  Int16: "Int16",
+  Int32: "Int32",
+  Int64: "Int64",
+  String: "String",
+  Boolean: "Boolean",
+  Bytes: "Bytes",
+};
+
+export type BaseTypes = typeof baseTypes;
+
+export type BaseType = keyof BaseTypes;
+
+export function isBaseType(type: string): type is BaseType {
```

Comment:
I'd expect an isSomething function to return a boolean

Reply:
Unless otherwise indicated, keeping it consistent with the AssemblyScript implementation: https://github.com/polywrap/monorepo/blob/prealpha/packages/schema/bind/src/bindings/wasm-as/types.ts#L16